src/elements/content-sidebar/SidebarContent.scss

```diff
@@ -1,8 +1,13 @@
 @import '../common/variables';
 
+$sidebarWidth: $sidebarContentWidth + 1px;
+$sidebarIncreasedWidth: $sidebarContentIncreasedWidth + 1px;
+$sidebarScrollContentWidth: 323px;
+$sidebarScrollContentIncreasedWidth: $sidebarScrollContentWidth + 40px;
+
 .bcs-content {
     position: relative;
-    width: 341px; // 1px extra to account for the margin left
+    width: $sidebarWidth; // 1px extra to account for the margin left
```

Comment:
Should we move this comment to the line where the 1px addition takes place (next to variable definition)?